Formula/twty.rb

```diff
@@ -0,0 +1,30 @@
+class Twty < Formula
+  desc "Command-line twitter client written in golang"
+  homepage "https://mattn.kaoriya.net/"
+  url "https://github.com/mattn/twty/archive/refs/tags/v0.0.11.tar.gz"
+  sha256 "d1ee544ff31a9a9488ff759da587baf927ab7c31b191b4b5bc010f36ecfb8188"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
```

Comment:
Actually we can update the ldflags to shrink the binary
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Reply:
Thanks!